1Password/pulumi-onepassword
# Rename python package name

## Summary

Having a dash (`-`) in the Python package name is not compliant to the Python module naming convention. Instead, we should use the underscore(`_`).

Resolves: #26 

## How to test

1. Checkout this branch:
   ```sh
   git checkout fix/python-package-name
   ```
2. In a python environment, install the local sdk:
   ```sh
   pip3 install -e $PWD/sdk/python
   ```
3. Make a python file with a content like this one (can be named `main.py`):
   ```python
   import pulumi
   import pulumi_onepassword as onepassword

   example = onepassword.get_item(
       vault=onepassword.data["onepassword_vault"]["example"]["uuid"],
       uuid=onepassword.onepassword_item["demo_sections"]["uuid"],
   )
   ```

4. Run the python file:
   ```sh
   python3 main.py
   ```

   - [ ] It should not fail with the following error:
     ```
     Cell In[1], line 1
       import pulumi-onepassword
                 ^
     SyntaxError: invalid syntax
     ```
